packages/cli/test/functional/test_site/expected/sub_site/nested_sub_site/index.html

```diff
@@ -11,18 +11,18 @@
   <meta name="generator" content="MarkBind 4.0.2">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title></title>
-  <link rel="stylesheet" href="../../markbind/css/bootstrap.min.css">
-  <link rel="stylesheet" href="../../markbind/fontawesome/css/all.min.css">
-  <link rel="stylesheet" href="../../markbind/glyphicons/css/bootstrap-glyphicons.min.css">
-  <link rel="stylesheet" href="../../markbind/css/octicons.css">
-  <link rel="stylesheet" href="../../markbind/material-icons/material-icons.css">
-  <link rel="stylesheet" href="../../markbind/css/codeblock-dark.min.css">
-  <link rel="stylesheet" href="../../markbind/css/markbind.min.css">
-  <script src="../../markbind/js/polyfill.min.js"></script>
-  <script src="../../markbind/js/vue.min.js"></script>
-  <script src="../../markbind/js/markbind.min.js"></script>
+  <link rel="stylesheet" href="/test_site/markbind/css/bootstrap.min.css">
```

Comment:
Is this a concern? I am thinking maybe it needs to read the files under the sub_site, which maybe somehow could be different from the outer site (maybe due to a change in markbind version).

Reply:
Nope, subsites also use the parent site's assets.

I understand the confusion; I don't think we're clearly communicating enough what subsites really are. (I was [confused](https://github.com/MarkBind/markbind/issues/1307) in the beginning as well, also I see it many times in https://github.com/MarkBind/markbind/pull/1870)

At the current its just a over the top way of saying: I will ensure your use of `{{ baseUrl }}` in subsites resolves correctly, if you try to _use_ a file in the subsite from the parent site.

"Use" being anything that requires resolving these links correctly:
- `<include/panel>`-ing it
- `{% include %}`-ing it
- probably more

I've come to think the status quo is fine (but the documentation clearly needs to be made clearer given all the confusion), but if you have any ideas please help to drive the discussion https://github.com/MarkBind/markbind/issues/1307! 🙌

cc @damithc

Reply:
In the CS2103 website, se-book has it's own assets which are used when it is on its own e.g., https://se-education.org/se-book/
But when the se-book is being used as a sub-site inside CS2103 website, the outer site's assets are being used for the se-book as well (I think).